### PR TITLE
Modifications in /help

### PR DESCRIPTION
**Pull request by spekss**
add /tour in te "moderator commans" and /modjoin in "Leader commands"

### CODE REVIEW

File: chat-commands.js [hunk 1]
```diff
@@ -3472,8 +3472,8 @@ exports.commands = {
 			this.sendReply("INFORMATIONAL COMMANDS: /data, /dexsearch, /movesearch, /groups, /faq, /rules, /intro, /formatshelp, /othermetas, /learn, /analysis, /calc (replace / with ! to broadcast. Broadcasting requires: + % @ * # & ~)");
 			if (user.group !== Config.groupsranking[0]) {
 				this.sendReply("DRIVER COMMANDS: /warn, /mute, /hourmute, /unmute, /alts, /forcerename, /modlog, /modnote, /lock, /unlock, /announce, /redirect");
-				this.sendReply("MODERATOR COMMANDS: /ban, /unban, /ip, /modchat");
-				this.sendReply("LEADER COMMANDS: /declare, /forcetie, /forcewin, /promote, /demote, /banip, /host, /unbanall");
+				this.sendReply("MODERATOR COMMANDS: /ban, /unban, /ip, /tour, /modchat");
```
Comment (panpawn) [inline chat-commands.js]: ``/tour`` is an alias, if we list it here, we should list it as ``/tournament``

Comment (spekss) [inline chat-commands.js]: ah y, I forgot this thanks

**Comment by Lionyx** [comment]
If one writes only "/tour", "/tours" or "/tournaments", it will just show the list of the ongoing tournaments, therefore it has nothing of a moderator command.
And if you wanted to mention the ability to create & manage tours (_/tour new_ etc), it depends on the room, the authority required to manage tours can be %, @ or #. In other words, it still aint a moderator command.
And for /modjoin, room owners can activate it (in groupchats at least) and so can players in battle rooms. This makes modjoin a bit controversial as a Leader command.

**Comment by spekss** [comment]
yeah, but player / roomowner are not a global ranks

**Comment by spekss** [comment]
But yeah, maybe change /tournament to /tournament create

**Comment by panpawn** [comment]
I think it's fine as-is

**Comment by Zarel** [comment]
There's no need for `/modjoin` to be documented; it's part of `/roomsettings`.

As for `/tour`, it's useful but I'm not sure it belongs in `/help` either.

**Comment by Zarel** [comment]
The bigger question is exactly where it goes.

**Comment by HoeenCoder** [comment]
Since tournament permissions vary by room (min %, max #) and since `/tour help` is basically a whole help command for tournaments, maybe just add a line stating _For help with tournaments use `/tournament help`_ Like the line for `/roomhelp`

**Comment by spekss** [comment]
well I removed the /modjoin, yeah hoeen, maybe just add /tournament help

**Comment by Zarel** [comment]
`/roomhelp` already mentions tour commands, right? No need to be redundant, then.

**Pull request closed without merging**